mail/rspamd/Portfile

```diff
@@ -74,11 +74,23 @@ depends_lib-append  port:fann \
 depends_run-append  port:hiredis \
                     port:redis
 
-# Match requirements of dependency hyperscan.
-supported_archs     x86_64
+variant hyperscan \
+    description {Use hyperscan instead of vectorscan.} {
+    depends_lib-replace \
+                    port:vectorscan \
+                    port:hyperscan
+    supported_archs x86_64
```

Comment:
Does `supported_archs` work in a variant?

Reply:
I'm not sure. What I do see is that `port install rspamd +hyperscan` on `arm64` compiles the entire dependency chain from scratch, which doesn't look like desired behavior to me, even if the ultimate binary can be linked to an `x86_64` binary and work [?]

I suggest using `known_fail` for this variant on `arm64`.

Reply:
If you mean that all of the dependencies get rebuilt with the universal variant, then that is working correctly. To build something (rspamd) for x86_64 on arm64, dependencies need to be built universal. I would recommend not adding code to the Portfile to subvert how MacPorts is intended to work.

Reply:
Please note that your original comment has already been addressed: `supported_arch`s is not used in a variant. 

Also, `hyperscan` only supports `x86_64`, which is the entire motivation of the request for this PR in https://trac.macports.org/ticket/69717.

To the best of my knowledge, this Portfile is configured correctly for the upstream ports and is ready to merge.